converters/toZigbee.js

```diff
@@ -514,12 +520,13 @@ const converters = {
 
                 await entity.command('lightingColorCtrl', 'moveColorTemp', payload, utils.getOptions(meta.mapped, entity));
 
-                // As we cannot determine the new brightness state, we read it from the device
+                // We cannot determine the color temperaturefrom the current state so we read it, because
+                // - Color mode could have been swithed (x/y or colortemp)
```

Comment:
I don't understand this change, if colorMode could change we should also ready the currentX/currentY?

Reply:
It's not a requirement that currentHue/currentSatuaturation/currentX/currentY gets updates.
Only what is the active color_mode 

I also observe this, e.g. innr will update colorTemperature when within range (or not when out of range), the tint bulb doesn't seem to do it.

Given we set colorTemperature, reading both should be fine and get us the current colorMode and bulb reported colorTemperature value.

Reply:
That does pose an interesting question, I guess the proper thing to do would be In both the color_temp and color converters... would be to read colorMode, check the result and then read either colorTemperature, currentHue + currentSaturation, or currentX + currentY.

That would guarantee we have the correct information from the bulb for the mode the bulb is set too. colorMode is a required value so barring manufacturer ignoring the ZCL (*sigh*) we should always be able to read colorMode if we have the ligthingColorCtrl cluster.

Reply:
I'm wondering if this converter should just return `return {state: {color_mode: constants.colorMode[2]}}`. This should avoid the read. 

I assume that when a bulb is e.g. in xy mode and you do a `colortemp_move` it switches to color temp mode thus the color_mode is always 'color_temp`.

Reply:
That could work too yes, I added it to the read because we are already doing one anyway, and it seems to still be one read request according to wireshark with 2 attribute instead of 1 requested. So I don't think this adds extra load on the network.

Reply:
Did you encounter bulbs which do not support the `colorMode` property?

Reply:
Not yet :) I tested my usual batch of Tint CWS E27, GU10, Trådfri WS GU10, CWS E27, WS FLOAT, WS GUNTARP, Innr RB 250 C, Osram A65 CWS E27.

Reply:
Good, let's assume for now that every bulb which supports `colortemp_move` also support `colorMode` (otherwise the colorMode could be wrong).

Reply:
<img width="993" alt="image" src="https://user-images.githubusercontent.com/379665/111526389-f8749e80-875e-11eb-8913-0d95df97eb75.png">

Until otherwise, I think this is a safe assumption given it is marked M without any additional notes.